test/commands/test-cli.js

```diff
@@ -0,0 +1,39 @@
+const mockStdin = require('mock-stdin');
+const { stdout, stderr } = require('stdout-stderr');
+const { expect } = require('chai');
+
+module.exports = async function testCli({
+  nock, env, inputs, command, outputs, errorOutputs,
+}) {
+  const previousEnv = process.env;
+  if (env) process.env = env;
```

Comment:
I don't think this `test-cli` file should be responsible of the env. This should be done in the test file, which should describe its own context and restore it right after the tests execution.

Reply:
It's the case, env come from the tests.